packages/@aws-cdk/aws-route53-targets/lib/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './api-gateway-domain-name';
 export * from './cloudfront-target';
 export * from './load-balancer-target';
+export * from './classic-load-balancer-target';
```

Comment:
can we make this alphabetical? - It's easier to search and find if there isn't a strong reason to append at the end.

Reply:
Done 👍